CONTRIBUTING.md

```diff
@@ -51,7 +51,7 @@ come up, including gaps in documentation!
 
 If you're a more experienced contributor, looking at unassigned issues in the next release milestone is a good way to find work that has been prioritized. For example, if the latest minor release is `v1.0`, the next release milestone is `v1.1`.
 
-Help and contributions are very welcome in the form of code contributions but also in helping to moderate office hours, triaging issues, fixing/investigating flaky tests, cutting releases, helping new contributors with their questions, reviewing proposals, etc.
+Help and contributions are very welcome in the form of code contributions but also in helping to moderate office hours, triaging issues, fixing/investigating flaky tests, being part of the [release team](docs/developer/release-team.md), helping new contributors with their questions, reviewing proposals, etc.
```

Comment:
nit: the link leads to 404 in the book.
I think this can be fixed either by moving the relase-team doc into the book or by changing the link to an absolute link (after merge)

Reply:
> I think this can be fixed either by moving the relase-team doc into the book 

even if I move it there is no way to make the link for both the book and the CONTRIBUTING.md file outside the book right? because if it's `developer/release-team.md` in the book that same link won't work in the repo tree.

If so the only way is to change it to an absolute link, which I can do after merge.

Reply:
You're right!

Reply:
Can we add a custom 404 page to netlify that redirects from `docs/<path>` to `<path>`? Then those links would work on both sites.